pkgs/by-name/nr/nrr/package.nix

```diff
@@ -5,12 +5,14 @@
 , darwin
 , pkg-config
 , libiconv
+, lto ? true
 , nrxAlias ? true
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "nrr";
   version = "0.9.4";
+  __structuredAttrs = true;
```

Comment:
It is not obvious from the commit message why this is necessary.

Reply:
It is mainly a matter of preference, and since I'm setting environment variables in this commit I thought it would be good to add this as well.